docs/static/breaking-changes.asciidoc

```diff
@@ -1,93 +1,256 @@
 [[breaking-changes]]
 == Breaking changes
 
-This section discusses the changes that you need to be aware of when migrating your application to Logstash {version}.
+This section discusses the changes that you need to be aware of when migrating your application to Logstash {version} from the previous major release of Logstash (2.x).
+
+[float]
+=== When to Upgrade
+
+Fresh installations can and should start with the same version across the Elastic Stack.
+
+Elasticsearch 5.0 does not require Logstash {version}. An Elasticsearch 5.0 cluster will happily receive data from a
```

Comment:
Be careful when using the version variable because this value will change with every release. Usually when you describe breaking changes, they are specific to a certain release. You don't want the version number to change when the release version changes. This comment applies to other places where you've used `{version}` in this document.